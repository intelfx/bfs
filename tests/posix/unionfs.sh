[[ "$UNAME" == *BSD* ]] || skip

cd "$TEST"
"$XTOUCH" -p lower/{foo,bar,baz} upper/{bar,baz/qux}

bfs_sudo mount -t unionfs -o below lower upper || skip
defer bfs_sudo umount upper

bfs_diff .
