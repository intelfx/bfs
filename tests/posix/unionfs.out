.
./lower
./lower/bar
./lower/baz
./lower/foo
./upper
./upper/bar
./upper/baz
./upper/baz/qux
./upper/foo
